lib/validation.tsx

```diff
@@ -1,6 +1,14 @@
 import React from "react";
-import { FormValues, DynamicFormProps, InnerFormProps, FormElement, Responses } from "./types";
-import { FormikProps } from "formik";
+import {
+  FormValues,
+  DynamicFormProps,
+  InnerFormProps,
+  FormElement,
+  Responses,
+  ValidationProperties,
+} from "./types";
+import { FormikProps, FormikValues } from "formik";
```

Comment:
Just a quick removal of `FormikValues` which isn't being used in this file.

Reply:
Good catch, thank you!